src/System.Buffers.Experimental/System/Buffers/ReadOnlyBytes.cs

```diff
@@ -264,7 +264,7 @@ public int CopyTo(Span<byte> buffer)
                     }
                     else
                     {
-                        segment.CopyTo(free);
+                        segment.Span.CopyTo(free);
```

Comment:
Why don't we have CopyTo on Memory<T>?

Reply:
Waiting for API review from this issue which adds a few instance and extension methods:
https://github.com/dotnet/corefx/issues/23862